templates/components.html

```diff
@@ -365,12 +365,14 @@ <h3 class="card-title main-title">
 {% macro rocketchat(rocketchat_id,chat_server) -%}
 {% if rocketchat_id %}
 {{ section("Chat") }}
-  <h6 align="left">To sign in to RocketChat, click the <b>Login via Auth0</b> button. You do not need to enter an email
-    and password. You will be asked to choose a chat nickname.</h6>
+  <h6 align="left"><p>To sign in to RocketChat, click the <b>Login via Auth0</b> button. You do not need to enter an email
+    and password. You will be asked to choose a chat nickname.</p>
+    <p>To open chat in a new browser tab click <a href="https://chat.chilconference.org" target="_blank">here</a>. </p>
```

Comment:
Thanks @kperros , I like this sentence structure and have followed it. When referring to the app itself I now use the phrase rocket.chat since that appears to be the official name for the app.